src/components/Signup.jsx

```diff
@@ -0,0 +1,156 @@
+                  value={name}
+                  onChange={(e) => {
+                    setName(e.target.value);
+                  }}
+                />
+              </div>
+            </label>
+
+            <label className="flex flex-col align-center gap-2 text-xl w-3/4 mt-4">
+              Email
+              <div className="flex flex-row items-center gap-4 rounded-md border-black border-2 h-10 p-2">
+                <MdEmail />
+                <input
+                  type="text"
+                  className="outline-none"
+                  value={email}
+                  onChange={(e) => {
+                    setEmail(e.target.value);
+                  }}
+                />
+              </div>
+            </label>
+
+            <label className="flex flex-col w-3/4 align-center text-xl gap-2 mt-4">
+              Password
+              <div className="flex flex-row items-center gap-4 rounded-md border-black border-2 h-10 p-2">
+                <MdVpnKey />
+                <input
+                  type={showPwd ? "text" : "password"}
+                  className="outline-none bg-white"
+                  value={password}
+                  onChange={(e) => {
+                    setPassword(e.target.value);
+                  }}
+                />
+                <div
+                  className="ml-auto"
+                  onClick={() => {
+                    setShowPwd((p) => !p);
+                  }}
+                >
+                  {showPwd ? <HiEye /> : <HiEyeOff />}
+                </div>
+              </div>
+            </label>
+
+            <label className="flex flex-col w-3/4 align-center text-xl gap-2 mt-4">
+              Confirm Password
+              <div className="rounded-md h-10 border-black border-2 flex flex-row items-center gap-4 p-2">
+                <MdVpnKey />
+                <input
+                  type={showConfirmPwd ? "text" : "password"}
+                  className=" outline-none bg-white"
+                  value={confirmPwd}
+                  onChange={(e) => {
+                    setConfirmPwd(e.target.value);
+                  }}
+                />
+                <div
+                  className="ml-auto"
+                  onClick={() => {
+                    setShowConfirmPwd((p) => !p);
+                  }}
+                >
+                  {showConfirmPwd ? <HiEye /> : <HiEyeOff />}
+                </div>
+              </div>
+            </label>
+
+            <div className="flex flex-row justify-between w-3/4 m-2">
+              <label>
+                <input
+                  type="checkbox"
+                  checked={isChecked}
+                  onChange={() => {
+                    setIsChecked((p) => !p);
+                  }}
+                  className="accent-red-200"
+                />
+                Accept terms & conditions
+              </label>
+              <p>Forgot password?</p>
+            </div>
```

Comment:
Here you can put it to the link to do login, for example 
Already have an account?